src/core/lib/gprpp/ref_counted.h

```diff
@@ -111,7 +180,7 @@ class RefCounted : public Impl {
  protected:
   GPRC_ALLOW_CLASS_TO_USE_NON_PUBLIC_DELETE
 
-  RefCounted() { gpr_ref_init(&refs_, 1); }
+  RefCounted() = default;
 
   // Note: Depending on the Impl used, this dtor can be implicitly virtual.
   ~RefCounted() {}
```

Comment:
Likewise, how about defaulting this instead of empty? That way we have a constexpr constructor and a trivial destructor, allowing objects of this nature to be constructed with global scope and lifetime. [stifles evil laugh]